postprocessor/src/main/java/com/azure/autorest/postprocessor/Postprocessor.java

```diff
@@ -237,4 +242,30 @@ private String getBaseDirectory() {
             Utils.deleteDirectory(tempDirWithPrefix.toFile());
         }
     }
+
+    private void handlePartialUpdate(Map<String, String> fileContents) {
+        logger.info("Begin handle partial update...");
+        // handle partial update
+        // currently only support add additional interface or overload a generated method in sync and async client
+        fileContents.replaceAll((path, generatedFileContent) -> {
+            if (path.endsWith(".java")) { // only handle for .java file
+                // get existing file path
+                String projectBaseDirectoryPath = new File(getBaseDirectory()).getParent();
+                URI existingFilePath = Paths.get(projectBaseDirectoryPath, path).toUri();
+                // check if existingFile exists, if not, no need to handle partial update
+                if (Files.exists(Paths.get(existingFilePath))) {
```

Comment:
Why do we need to get the URI? Can we just do the following?
```java
if (Files.exists(Paths.get(projectBaseDirectoryPath, path))) 
```

Reply:
updated